recipes/bbhash/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "bbhash" %}
+{% set version = "0.5.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bbhash-{{ version }}.tar.gz
+  sha256: 7e84dc10962fe09c666ff2eddad02c08b8dd6f83fb73f7023dbf93d7dea30f1d
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - cython >=0.29.21
```

Comment:
I think only the compilers are needed in the `build` section.

Reply:
cython is also needed to create the .cpp files.

Reply:
That's handled by what's in `host`. Let's remove everything but the compilers in the `build` section.